### PR TITLE
[HMC-18] - Slicing FAQ Accordion

### PR DESCRIPTION
**Pull request by adzaky**
## Description :

- Create FAQ's Accordion
- Add FAQ's Constant and Type

## Screenshots / Video Record:

https://github.com/hammer-code/lms-fe/assets/92772395/c62aec68-dbb0-4427-bf56-6bf998ceddc2

### CODE REVIEW

File: src/features/Main/components/AccordionFAQ/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+'use client';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger
+} from '@/components/ui/accordion';
+import { Card, CardContent } from '@/components/ui/card';
+import { FAQ } from '@/constants/faq';
+import React, { FC } from 'react';
```
Comment (vickyadrii) [inline src/features/Main/components/AccordionFAQ/index.tsx:10]: keknya yang `React` nda kepake, nnti remove kalo nda kepake yaaap, tpi kalo kepake, abaikan saja okey

Comment (adzaky) [reply src/features/Main/components/AccordionFAQ/index.tsx]: My bad, done kak @vickyadrii

**Review by vickyadrii — APPROVED** [review]

File: src/features/Main/components/AccordionFAQ/index.tsx (hunk 1 above)
Comment (vickyadrii) [reply src/features/Main/components/AccordionFAQ/index.tsx]: mantepp. approved @adzaky

**Comment by mpsalunggg** [comment]
@adzaky i think u should record when responsive

**Pull request merged**